tools/testing/target_determination/heuristics/interface.py

```diff
@@ -159,34 +254,37 @@ def _print_tests(label: str, tests: StringTuple) -> None:
                 if test in tests:
                     print(f"  {test}")
 
-        for relevance_group, tests in enumerate(self._test_priorities):
+        for relevance_group, tests in self._traverse_priorities():
             _print_tests(f"{Relevance(relevance_group).name.title()} Relevance", tests)
 
     def _get_test_relevance_group(self, test_name: str) -> Relevance:
```

Comment:
nit but probably specify that this is the highest relevance for any subset of this test, also probably for the below functions

Reply:
Good call